api/v1beta1/flinkcluster_types.go

```diff
@@ -567,6 +571,9 @@ type JobStatus struct {
 	// Last savepoint trigger ID.
 	LastSavepointTriggerID string `json:"lastSavepointTriggerID,omitempty"`
 
+	// Last successful or failed savepoint operation timestamp.
```

Comment:
What if the operation is still in progress?

Reply:
I'll change the comment here. This flow is still not 100% bug proof, I think more work needs to be done on savepoint flow.

Reply:
Fixed the doc there